code/game/objects/structures/ghost_pods/human.dm

```diff
@@ -123,7 +124,9 @@
 	if(allow_appearance_change)
 		H.change_appearance(APPEARANCE_ALL, H, check_species_whitelist = 1)
 
-	visible_message("<span class='aliem'>\The [src] [pick("gurgles", "seizes", "clangs")] before releasing \the [H]!</span>")
+//	visible_message("<span class='aliem'>\The [src] [pick("gurgles", "seizes", "clangs")] before releasing \the [H]!</span>")
```

Comment:
If you don't plan on doing something with this commented out bit, why not just remove it.

Reply:
You ever just type something out, forget about it, then restart your browser? Yeah I did that like 8 times on this aaaaaaa sorry.

I left it in comment, mostly as reference, but if anyone wanted to go back and restore them to being visible, it's still there.